### DISCUSSION TITLE
2008/07/20/%E5%A6%82%E4%BD%95%E5%AD%A6%E4%B9%A0-labview

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#234] [Announcements]
# 2008/07/20/%E5%A6%82%E4%BD%95%E5%AD%A6%E4%B9%A0-labview

根据我自己的观察，学习LabVIEW一般有以下三种方式：系统型学习方法、探索型学习方法和目标驱动型学习方法。这三种方法之间并不矛盾，可以在不同的时段使用不同的方法。每个人可以根据自己的个性特点和所处环境选择一个适合自己的学习方案。

https://blog.qizhen.xyz/2008/07/20/%E5%A6%82%E4%BD%95%E5%AD%A6%E4%B9%A0-labview

**Comment by ruanqizhen** [comment]
之前博客下的留言：

杰
2008年07月20日 22:45 编辑
“我见过几个工程师在项目中遇到了难题，于是来报名参加LabVIEW的培训课程，以为上完课可以解决自己的问题。但实际上完全误解了培训课程的意义，培训课程是为了帮助那些想要系统学习LabVIEW知识的人，而不专注于任何一个具体问题。”
 
PLS Check the information at NI official website regarding the training program, you can understand why those engineers thought the training could help them to solve all the problems. For example:
经济、高质的专业训练课程均由知识丰富的应用工程师讲授，他们可以和您讨论应用程序的解决方案，帮助您的项目快速起步。
帮助您开发大型应用程序，并能轻松进行维护及重复使用.
 

回复
杰
2008年07月20日 22:38 编辑
哈哈，小阮这个分析的好玩。
我首先是目标驱动型，完成任务之后变成了探索型，现在是系统学习型，一个VI一个VI的看，用用满好玩的，呵呵。
不过LabVIEW涵盖的东西和应用范围过广，还是只能学习纯软件的东西，很多工具包如果不是确实要用，基本上不会碰的。

回复
